src/Microsoft.DotNet.XHarness.Common/Execution/ProcessManager.cs

```diff
@@ -135,44 +136,49 @@ static async Task KillTreeAsyncInternal(int pid, Action<int, int> kill, Func<ILo
                         diagnostics: false);
                 }
 
-                foreach (var diagnose_pid in pids)
+                const string lldbPath = "/usr/bin/lldb";
```

Comment:
@akoeplinger should we just do `lldb` and let PATH handle it?

Reply:
Yeah. I'd also not check for existence of lldb and just make a catch-all so any exception from launching lldb is caught and ignored.

Reply:
Fixed